search/templates/cosmos/searchresults.html

```diff
@@ -50,6 +50,31 @@ <h4 class="card-title">
                                 aria-haspopup="true" aria-expanded="false">
                             Solutions
                         </button>
+                    <div>
+                    <div >
+                        <ul class="sol_tab nav nav-tabs">
+                            <li class="active">
+                     <a href= "#tab1default" style="color: #1b1e21" data-toggle="tab">
+                         <img src="../static/images/wikipedia.png"/>
+                     </a>
+</li>
+                            <li>
+                     <a href="#tab2default" style="color: #1b1e21" data-toggle="tab"><img src="../static/images/stack-overflow.png"/></a>
+</li>
+</ul>
+
+                    </div>
+                    <div class="tab-content">
+                        <div class="tab-pane fade in active" id="tab1default">
+                            <p><a href="{{ res_wiki.url }}" target="blank"/>{{ res_wiki.heading }}</p>
+                        </div>
+                        <div class="tab-pane fade" id="tab2default">
```

Comment:
The ids of those dropdown need be difference.

Reply:
Will you please tell about which dropdown you are talking, because as of now there is no dropdown in that script.

Reply:
Sorry, what I want to say is `collapse`, you could try to clicking wiki/stack on second result, then you get it.